File: warehouse/kafka/schemas/EnerginetProductionAndExchangeSchema.js

exports.Enum = Enum;
exports = class EnerginetProductionAndExchangeSchema {
    constructor();
    

    /**
    *
    * @param { string } MINUTES5_DK
    * @param { string } PRICE_AREA
    * @param { number } PRODUCTION_LT_100
    * @param { number } PRODUCTION_GE_100
    */
    constructor(
        MINUTES5_DK,PRICE_AREA,PRODUCTION_LT_100,PRODUCTION_GE_100
    ){
        this.MINUTES5_DK=MINUTES5_DK;
        this.PRICE_AREA=PRICE_AREA;
        this.PRODUCTION_LT_100=PRODUCTION_LT_100;
        this.PRODUCTION_GE_100=PRODUCTION_GE_100;
    }


    
    /**
    * Copy a js object into this.
    * @param {*} jsonObject the js object 
    */
    copyInto(jsonObject){
        if(jsonObject.MINUTES5_DK){
            this.MINUTES5_DK=jsonObject.MINUTES5_DK;
        }
        if(jsonObject.PRICE_AREA){
            this.PRICE_AREA=jsonObject.PRICE_AREA;
        }
        if(jsonObject.PRODUCTION_LT_100){
            this.PRODUCTION_LT_100=jsonObject.PRODUCTION_LT_100;
        }
        if(jsonObject.PRODUCTION_GE_100){
            this.PRODUCTION_GE_100=jsonObject.PRODUCTION_GE_100;
        }
    }
}

File: spark-submitter/src/main/java/dk/sdu/datascience/kafka/structure/schemas/EnerginetProductionAndExchangeSchema.java

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package dk.sdu.datascience.kafka.structure.schemas;

import com.google.gson.annotations.SerializedName;
import java.util.List;
/**
 *
 * @author Async api generated
 */
public class EnerginetProductionAndExchangeSchema {
    
        
    private String MINUTES5_DK;

    /**
    * @return MINUTES5_DK
    */
    public String getMINUTES5_DK() {
        return MINUTES5_DK;
    }

    /**
    * @param MINUTES5_DK to set
    */
    public void setMINUTES5_DK(String MINUTES5_DK) {
        this.MINUTES5_DK = MINUTES5_DK;
    }

        
    private String PRICE_AREA;

    /**
    * @return PRICE_AREA
    */
    public String getPRICE_AREA() {
        return PRICE_AREA;
    }

    /**
    * @param PRICE_AREA to set
    */
    public void setPRICE_AREA(String PRICE_AREA) {
        this.PRICE_AREA = PRICE_AREA;
    }

        
    private Double PRODUCTION_LT_100;

    /**
    * @return PRODUCTION_LT_100
    */
    public Double getPRODUCTION_LT_100() {
        return PRODUCTION_LT_100;
    }

    /**
    * @param PRODUCTION_LT_100 to set
    */
    public void setPRODUCTION_LT_100(Double PRODUCTION_LT_100) {
        this.PRODUCTION_LT_100 = PRODUCTION_LT_100;
    }

        
    private Double PRODUCTION_GE_100;

    /**
    * @return PRODUCTION_GE_100
    */
    public Double getPRODUCTION_GE_100() {
        return PRODUCTION_GE_100;
    }

    /**
    * @param PRODUCTION_GE_100 to set
    */
    public void setPRODUCTION_GE_100(Double PRODUCTION_GE_100) {
        this.PRODUCTION_GE_100 = PRODUCTION_GE_100;
    }


}
